scripts/faked-debug.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
```

Comment:
I'ld appreciate some comment here about why this is needed. Otherwise nobody will remember in a year.